blas/tpls/KokkosBlas_Host_tpl.hpp

```diff
@@ -90,18 +90,11 @@ struct HostBlas {
                    KK_INT incx, const T *y, KK_INT incy, T *a, KK_INT lda);
 
   template <typename tAlpha>
```

Comment:
I suppose we could infer this from the `std::complex<T>::value_type`, but I'm just looking for a minimum viable change here.

Reply:
(I'll mark this as resolved, assuming there are no objections.)

Reply:
To follow the BLAS specification exactly we probably want the following:
```c++
static void her(const char uplo, KK_INT n, const mag_type alpha, const T *x,
                KK_INT incx, T *a, KK_INT lda);
```
although I am not sure is this would work without a few more changes to the types in the definition of this kernel.
That's a bigger question though because I believe we do not follow the BLAS specification in other function calls.